docs/serving/samples/knative-routing-go/README.md

```diff
@@ -174,8 +191,26 @@ You should see: `Login Service is called !`
 kubectl apply --filename docs/serving/samples/knative-routing-go/routing.yaml
 ```
 
+If you have already had an existing domain name for your service, please replace all
+the presences of "example.com" in routing.yaml with "<your domain name>" for
```

Comment:
maybe:
"if you have configured a custom domain name for your service, please replace all mentions of ..."

Reply:
Done. thx.